manifests/install.pp

```diff
@@ -42,6 +42,49 @@
       path    => '/bin'
     }
   }
+
+  elsif $opendaylight::install_method == 'deb'{
+    
+    include apt
+    
+    apt::backports { 'jessie-backports':
+      comment  => 'This is Debian jessie-backports to install Java 8',
+      location => 'http://httpredir.debian.org/debian',
+      release  => 'jessie-backports',
+      repos    => 'main',
+      notify   => Exec['apt_update'],
```

Comment:
ubuntu@ubuntu-xenial:~$ sudo systemctl status opendaylight
● opendaylight.service
   Loaded: not-found (Reason: No such file or directory)
   Active: inactive (dead)